rakefile

```diff
@@ -290,7 +290,7 @@ rototiller_task :soak do
   ENV['BEAKER_PRESERVE_HOSTS'] = ENV['BEAKER_PRESERVE_HOSTS'] || 'always'
   ENV['PUPPET_SCALE_CLASS'] = ENV['PUPPET_SCALE_CLASS'] || 'role::by_size::large'
   ENV['PUPPET_GATLING_SCALE_TUNE'] = ENV['PUPPET_GATLING_SCALE_TUNE'] || 'true'
-  ENV['ABS_AWS_REAP_DAYS'] = ENV['ABS_AWS_REAP_DAYS'] || '20'
+  ENV['ABS_AWS_REAP_DAYS'] = ENV['ABS_AWS_REAP_DAYS'] || '30'
```

Comment:
Need some kind of doc update about this new env var.  I think the old one wasn't documented either though.  So probably a small section in the readme about both of these variables.  And maybe a mention in the soak section of setting it to 30 days...  helps someone think about if they want it larger so they don't lose their instances when the least expect it.